Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -9917,6 +9918,1640 @@
+										},
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													"pm.environment.set(\"supplied:studentSpecialEducationProgramAssociations:beginDate\", \"2021-09-07\");"
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "POST",
+										"header": [],
+										"body": {
+											"mode": "raw",
+											"raw": "{\r\n    \"educationOrganizationReference\": {\r\n        \"educationOrganizationId\": 255901\r\n    },\r\n    \"programReference\": {\r\n        \"educationOrganizationId\": 255901,\r\n        \"programName\": \"Special Education\",\r\n        \"programTypeDescriptor\": \"uri://ed-fi.org/ProgramTypeDescriptor#Special Education\"\r\n    },\r\n    \"studentReference\": {\r\n        \"studentUniqueId\": \"604872\"\r\n    },\r\n    \"beginDate\": \"{{supplied:studentSpecialEducationProgramAssociations:beginDate}}\",\r\n    \"ideaEligibility\": true,\r\n    \"iepBeginDate\": \"2021-09-01\",\r\n    \"iepReviewDate\": \"2021-08-27\",\r\n    \"lastEvaluationDate\": \"2021-08-23\",\r\n    \"schoolHoursPerWeek\": 10.00,\r\n    \"specialEducationHoursPerWeek\": 25.00,\r\n    \"specialEducationSettingDescriptor\": \"uri://ed-fi.org/SpecialEducationSettingDescriptor#Inside regular class less than 40% of the day\",\r\n    \"disabilities\": [\r\n        {\r\n            \"disabilityDescriptor\": \"uri://ed-fi.org/DisabilityDescriptor#Autism Spectrum Disorders\"\r\n        }\r\n    ]\r\n}",
+											"options": {
+												"raw": {
+													"language": "json"
+												}
+											}
+										},
+										"url": {
+											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/studentSpecialEducationProgramAssociations",
+											"host": [
+												"{{ApiBaseUrl}}"
+											],
+											"path": [
+												"data",
+												"v3",
+												"ed-fi",
+												"studentSpecialEducationProgramAssociations"
+											]
+										}
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "Read Requests",
+							"item": [
+								{
+									"name": "The Read content type only excludes certain properties",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													"pm.test(\"Status code is 200\", () => {\r",
+													"    pm.expect(pm.response.code).to.equal(200);\r",
+													"});\r",
+													"\r",
+													"const originalEntity = pm.environment.get(\"known:school\");\r",
+													"const profileEntity = pm.response.json();\r",
+													"const keys = pm.environment.get(\"known:school:PrimaryKeys\").concat(\"_etag\", \"_lastModifiedDate\");\r",
+													"const profileExcludeKeys = [\r",
+													"    \"nameOfInstitution\",\r",
+													"    \"operationalStatusType\",\r",
+													"    \"operationalStatusDescriptor\",\r",
```

Comment:
this property was not included in the specflow test, should be removed or was missing from the original test?

Reply:
`OperationalStatusType` is mentioned in the specflow test, but that property is not listed in the Swagger Model. I swapped it with `OperationalStatusDescriptor`.